src/mesh/mod.rs

```diff
@@ -0,0 +1,1481 @@
+use std::cmp;
```

Comment:
Just a question - wouldn't it be cleaner to keep the logic in `src/mesh/mesh.rs` and pub import it here?

Reply:
Sure, we can do that.

Reply:
Haha, turns out there is a clippy lint against just this :smile: Let's leave it as it then. We do the same with renderer and interpreter.